Library/Homebrew/services/formula_wrapper.rb

```diff
@@ -170,14 +184,14 @@ def owner
 
       sig { returns(T::Boolean) }
       def pid?
-        pid.present? && pid.positive?
+        pid.present? && T.must(pid).positive?
```

Comment:
```suggestion
        pid = self.pid
        pid.present? && pid.positive?
```
and similarly elsewhere